plutus-core/testlib/PlutusCore/Generators/PIR.hs

```diff
@@ -0,0 +1,9 @@
+module PlutusCore.Generators.PIR (module X) where
```

Comment:
`PlutusCore.Generators.PIR` :thinking: 
`PlutusIR.Generators`?

Reply:
Or the type stuff should be in `PlutusCore` but the term stuff should be in `PlutusIR`. Otherwise this is a very weird module structure.